### PR TITLE
Update napari viewer API and add viewer tests 

### PR DESCRIPTION
**Pull request by ieivanov**

### CODE REVIEW

**Comment by ieivanov** [comment]
Let's add documentation on the viewer API [here](https://pycro-manager.readthedocs.io/en/latest/acq_overview.html) before merging this PR. I'll work on that.

**Comment by henrypinkard** [comment]
Sounds good!

**Comment by ieivanov** [comment]
This looks good to me, @henrypinkard feel free to make changes to the documentation

File: pycromanager/acquisitions.py [hunk 1]
```diff
@@ -258,8 +259,12 @@ def __init__(
             startSequence. A common use case for this hook is when one want to send TTL triggers to the camera from an
             external timing device that synchronizes with other hardware. Accepts either one argument (the current
             acquisition event) or two arguments (current event, event_queue)
-        show_display : bool or str
-            If True, show the image viewer window. If False, show no viewer. If 'napari', show napari as the viewer
+        show_display : bool
+            If True, show the image viewer window. If False, show no viewer.
+        napari_viewer : napari.Viewer
+            Provide a napari viewer to display acquired data in napari (https://napari.org/) rather than the built-in
+            NDViewer. None by default. Data is added to the 'pycromanager preview' layer, which may be pre-configured by
```
Comment (henrypinkard) [inline pycromanager/acquisitions.py:266]: Maybe worth changing this away from "preview" to "data"? "view"? or something else?

File: docs/source/viewers.rst [hunk 2]
```diff
@@ -0,0 +1,59 @@
+.. _viewers:
+
+**************************
+Live preview
```
Comment (henrypinkard) [inline docs/source/viewers.rst:4]: I'd suggest changing the terminology from "preview" to either "viewer" or "display" for consistency with the API

Comment (henrypinkard) [reply docs/source/viewers.rst:4]: Or "Image viewer" or "Image display"

**Comment by henrypinkard** [comment]
Besides these small comments, looks good to merge!

File: pycromanager/acquisitions.py (hunk 1 above)
Comment (ieivanov) [reply pycromanager/acquisitions.py:266]: How about `pycromanager acquisition`?

**Comment by ieivanov** [comment]
Thanks @henrypinkard, take a look again at the last commit to see if you like this nomenclature

File: pycromanager/acquisitions.py (hunk 1 above)
Comment (henrypinkard) [reply pycromanager/acquisitions.py]: Sounds good!

**Comment by henrypinkard** [comment]
Looks great!

**Pull request merged**